newswires/client/src/catcodes-lookup.ts

```diff
@@ -0,0 +1,28 @@
+import { lookupTables } from './category-code-lookup-tables';
+
+const qCodePrefixesLookup = {
+	medtop: ['IPTC MediaTopics'],
+	subj: ['IPTC NewsCodes', 'IPTC MediaTopics'],
+	a1312cat: ['A1312 CatCodes'],
+	n2000: ['N2000 Codes'],
+	n2: ['N2000 Codes'],
+};
+
+export function lookupCatCodesWideSearch(catCode: string): string[] {
```

Comment:
We'd want to refine this at some point, but feels useful to try and err on the side of matching too many at this early-ish stage of trying to understand the data?